src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -4192,6 +4192,150 @@ def test_aks_nodepool_add_with_artifact_streaming(
             checks=[self.is_empty()],
         )
 
+    @live_only()  # live only is required for test environment setup like `az login`
```

Comment:
This is not a suitable reason. The scenario test (based on vcr test) in azure-cli runs in replay mode in built-in CI and live mode in the custom CI (aks live test pipeline). To run the test in replay mode, it will require that the corresponding cassette/recording file exists, otherwise it will fall back and try to execute in live mode, thus encountering the error you described here.

I removed the @live_only(), [queued the test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=88782952&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=784b8444-aafc-5d5f-d55c-dcb731c2d713) and there's no problem to run the test in replay mode with the cassette files. So please remove the tag and commit the recording files (auto generated after running test cases in live mode successfully, could find them from pipeline artifact).